tutorials/roofit/rf408_RDataFrameToRooFit.C

```diff
@@ -67,8 +67,13 @@ void rf408_RDataFrameToRooFit()
   // - the constructor arguments for RooDataSet (they follow the same syntax as the usual RooDataSet constructors)
   // - the column names that RDataFrame should fill into the dataset
   //
+  // The returned object is not yet a RooDataSet, but an RResultPtr that will be
+  // lazy-evaluated once you call GetValue() on it. But we will only do this later
```

Comment:
it's a good comment but maybe we could improve it a bit, for example add a link to the docs describing laziness in more detail: https://root.cern/doc/master/classROOT_1_1RDataFrame.html#actions so that people really understand what it is (if they've never seen this before)